Makefile

```diff
@@ -95,7 +96,7 @@ build:
 
 .PHONY: setup
 setup: build
-	#docker-compose scale elasticsearch=3
+	docker-compose up -d
```

Comment:
This already starts the service. I would not expect this from `setup`. Does this belong into `start`